pages/common/tlmgr-option.md

```diff
@@ -0,0 +1,28 @@
+# tlmgr option
+
+> TeX Live settings manager.
+> More information: <https://www.tug.org/texlive/tlmgr.html>.
+
+- List all TeX Live settings:
+
+`tlmgr option showall`
+
+- List all currently set Tex Live settings:
+
+`tlmgr option show`
+
+- Print all TeX Live settings as JSON array:
```

Comment:
```suggestion
- Print all available TeX Live settings as JSON array:
```

Reply:
Shouldn't it also be `as **a** JSON array`

Reply:
Maybe. `as JSON` would be the best imo, but if you want to keep the explicit mention of the `array`, I agree.

Reply:
But it's not only `available` ones, but all existing ones. And I prefer `as JSON`.